File: android/app/src/main/java/com/lightningkite/kwifttemplate/xml/RowTextXml.kt
//
// RowTextXml.swift
// Created by Kwift XML Android
//
package com.lightningkite.kwifttemplate.xml

import android.widget.*
import android.view.*
import com.lightningkite.kwift.views.actual.*
import com.lightningkite.kwift.views.shared.*
import com.lightningkite.kwifttemplate.R

class RowTextXml {

    lateinit var label: TextView

    fun setup(dependency: ViewDependency): View {
        val view = LayoutInflater.from(dependency.context).inflate(R.layout.row_text, null, false)
        label = view.findViewById<TextView>(R.id.label)
        return view
    }
}

File: ios/com/lightningkite/kwifttemplate/xml/layout/RowTextXml.swift
//
// RowTextXml.swift
// Created by Kwift XML
//

import UIKit
import FlexLayout
import PinLayout

class RowTextXml {
    
    weak var label: UILabel!
    
    func setup(_ dependency: ViewDependency) -> UIView {
        return { () -> UILabel in 
            let view = UILabel(frame: .zero)
            self.label = view
            view.layoutMargins = UIEdgeInsets(top: 8, left: 8, bottom: 8, right: 8)
            view.numberOfLines = 0
            view.font = UIFont.get(size: 16, style: [])
            view.textColor = UIColor(argb: 0xFF222222)
            return view
        }()
        
    }
}